protected/controllers/ModuleController.php

```diff
@@ -85,6 +84,7 @@ public function actionSaveModule()
         $titleUa = Yii::app()->request->getPost('titleUA', '');
         $titleRu = Yii::app()->request->getPost('titleRU', '');
         $titleEn = Yii::app()->request->getPost('titleEN', '');
+
         if(Module::model()->addNewModule($_POST['idCourse'], $titleUa, $titleRu, $titleEn, $_POST['lang'])){
             $count=count(Yii::app()->db->createCommand("SELECT DISTINCT id_module FROM course_modules WHERE id_course =" . $_POST['idCourse']
             )->queryAll());
```

Comment:
Can you also take a look to this strange construction? It is not safe and wierd-looking. I guess here we try to determine number of modules in course